tests/cases/classes/method-type-long

```diff
@@ -0,0 +1,97 @@
+---DESCRIPTION---
+Shorthand types must be used
+---FILENAME---
+Foo.php
+---CONTENTS---
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor;
+
+class Foo
+{
+    /** @var integer */
+    public $foo;
+
+    /** @var boolean */
+    public $bar;
+
+    /** @var array<integer, boolean> */
+    public $baz;
+
+    /**
+     * @param integer $quux
+     *
+     * @return integer
+     */
+    public function qux($quux)
```

Comment:
Didn't use `TypeHintDeclaration` to force these to be native types, but in this case it doesn't `extend`/`implement` anything. Wonder if we can get it to work in that case? Would mean it could create breaking changes though...

Reply:
https://github.com/slevomat/coding-standard/pull/628 got closed pretty quickly. Might become an option in the future.